### PR TITLE
[1983] Add unauthorized route, action and page

### PR DESCRIPTION
**Pull request by dankmitchell**
### Context
Unauthorized users

### Changes proposed in this pull request
Add `/401` page and route as per manage courses ui

### Guidance to review
🚢

### CODE REVIEW

**Comment by dankmitchell** [comment]
[Port "We don’t know which organisation you’re part of" to Rails](https://trello.com/c/Uqtuv3GA/1983-port-we-dont-know-which-organisation-youre-part-of-to-rails)

**Comment by defong** [comment]
image

File: app/controllers/sessions_controller.rb [hunk 1]
```diff
@@ -42,7 +42,7 @@ def signout
   end
 
   def failure
-    redirect_to "/401"
+    redirect_to unauthorized_path
```
Comment (fofr) [inline app/controllers/sessions_controller.rb]: Should this be redirecting or just responding with a 401?

Comment (dankmitchell) [reply app/controllers/sessions_controller.rb]: This is the frontend so yes, we should redirect.

Comment (fofr) [reply app/controllers/sessions_controller.rb]: We respond with a 404 and show page not found for that URL, we don't redirect to `/404`.
Why isn't this the same?

**Pull request merged**